storybook/.storybook/manager.ts

```diff
@@ -6,10 +6,11 @@ const theme = create({
   base: 'light',
   brandTitle: 'Fusion React Components',
   brandUrl: 'https://github.com/equinor/fusion-react-components',
+  brandImage: 'https://avatars.githubusercontent.com/u/525862?s=48&v=4',
```

Comment:
Consider Fusion logo and not equinor?

Reply:
Removed logo, more descriptive to have the title there.